stave/models.py

```diff
@@ -1414,3 +1437,16 @@ class Meta:
             ),
             # models.CheckConstraint(check=Q(application=F("question__application")), name="response_app_and_question_app_match")
         ]
+
+
+class GameHistory:
+    game: Game
+    user: User
+    role: Role
+    secondary_role: Role | None
+
+    def __init__(self, game: Game, user: User, role: Role, secondary_role: Role | None):
```

Comment:
Python/django isn't my day-to-day. Is there a more idiomatic way to do this?

Reply:
No, this is fine - you can do `secondary_role: Role | None = None` to default it so that callers do not have to specify, but in this case it seems fine to just specify it.